galaxy/Dockerfile

```diff
@@ -65,8 +68,6 @@ RUN mkdir -p $GALAXY_CONFIG_DIR $GALAXY_CONFIG_DIR/web && chown -R $GALAXY_USER:
 RUN su $GALAXY_USER -c "cp $GALAXY_ROOT/config/galaxy.ini.sample $GALAXY_CONFIG_FILE"
 ADD ./reports_wsgi.ini.sample $GALAXY_CONFIG_DIR/reports_wsgi.ini
 ADD ./htpasswd /etc/nginx/htpasswd
-ADD ./servers.ini $GALAXY_CONFIG_DIR/servers.ini
-RUN cat $GALAXY_CONFIG_DIR/servers.ini >> $GALAXY_CONFIG_FILE && rm $GALAXY_CONFIG_DIR/servers.ini
```

Comment:
Have a look at: https://travis-ci.org/bgruening/docker-galaxy-stable/builds/93799734

Only three tests are failing and this is due to a bioblend timeout not related to this issue. I can see handlerX.log files in `/home/galaxy/`. So I guess it's working :)

Any other thing I can test?